invokeai/backend/requantize.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict
+
+import torch
+from optimum.quanto.quantize import _quantize_submodule
+
+# def custom_freeze(model: torch.nn.Module):
+#     for name, m in model.named_modules():
+#         if isinstance(m, QModuleMixin):
+#             m.weight =
+#             m.freeze()
+
+
+def requantize(
+    model: torch.nn.Module,
+    state_dict: Dict[str, Any],
+    quantization_map: Dict[str, Dict[str, str]],
+    device: torch.device = None,
+):
```

Comment:
Add docs indicating where this was copied from and what was modified.

Reply:
[38c2e78](https://github.com/invoke-ai/InvokeAI/pull/6739/commits/38c2e7801f6ca5de7ee300fce0bcb79a46a52562)